internal/backends/database.go

```diff
@@ -109,6 +124,10 @@ type CreateCollectionParams struct {
 func (dbc *databaseContract) CreateCollection(ctx context.Context, params *CreateCollectionParams) error {
 	defer observability.FuncCall(ctx)()
 
+	must.BeTrue(params.CappedSize >= 0)
```

Comment:
I mean this will be validated in handler layer.

```suggestion
	must.BeTrue(params.CappedSize > 0)
```

Reply:
`CappedSize == 0` means that the collection is not capped.
There is no separate boolean flag because the zero value is useful already.

Reply:
Got it, I misunderstood :see_no_evil: